packages/fury-adapter-apib-serializer/test/filters-test.js

```diff
@@ -0,0 +1,55 @@
+const { expect } = require('chai');
+const { Fury } = require('fury');
+const { bodyOnly } = require('../lib/filters');
+
+const fury = new Fury();
+const { minim: namespace } = fury;
+
+describe('filters', () => {
+  describe('bodyOnly', () => {
```

Comment:
ALL of these test would fail in the old implementation because we the old implementation did not return boolean state and instead could result in values being returned as "truthy" (`return payload.messageBody` etc). I've altered the implementation to be simpler, and to always return boolean.